src/home.js

```diff
@@ -0,0 +1,23 @@
+const home = (content) => {
```

Comment:
## Improve the layout of the page. 
- [ ] The layout of the page is not loading the styles as expected. Please note that this should be working before we proceed. Below 👇🏾 is what we currently have.

![restaurant-page](https://user-images.githubusercontent.com/22345961/110133467-645d1b80-7ddd-11eb-9caa-8c1651cbacc6.png)

Reply:
## Required Change
From the last review, images and styles are still not loading when a user runs the project locally. 
Having studied the issue in-depth, I noticed that images were not being referenced properly as in the case of using them in different modules. Kindly take note of the directory in which the images are residing. In our case, a safe way to reference our images would be to have them prepended with a period before referencing the files to have javascript get the exact file you are referencing.

-  Take for instance for the image bit. `image.src = '../images/meal.png';` is more preferable than `image.src = '/images/meal.png';` as the former is more accurate than the latter since it is getting the right directory and the exact file at the same time since the `images` directory appears to be one level directory above from where they are getting referenced.
- As for the styling, you also need to indicate to webpack how to minimize them in the dist directory. That was the main purpose of the `webapck` to minimize files for us.

Kindly work on these changes then submit a request for review. Be sure to reach out if you feel stuck with the changes, thank you

Reply:
## Update on review changes
✅ Thank you for working on the images bit. The images are now getting displayed.
![restaurant-page](https://user-images.githubusercontent.com/22345961/110209442-b49bb280-7e9d-11eb-9296-dc85a755c8b8.png)

### Let's now work on the styling bit.
 - [ ] Please see my review on this in the `dist/index.html` file.